### ISSUE TITLE
Bump Jadu/SelectWoo to jadu-1.1.5

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#1446]
The latest version of Jadu/SelectWoo contains various ARIA related improvements.

Update to https://github.com/jadu/selectWoo/releases/tag/jadu-1.1.5 

Note: there should be no visual or functional differences versus current version. Improvements are only perceivable in screen readers and accessibility tools.
